pkg/pillar/cmd/upgradeconverter/upgradeconverter.go

```diff
@@ -34,6 +34,10 @@ var preVaultconversionHandlers = []ConversionHandler{
 		description: "Move ConfigItemValueMap from /persist/config to /persist/status",
 		handlerFunc: moveConfigItemValueMap,
 	},
+	{
+		description: "Move any json configuration files from /config/GlobalConfig to /persist/status",
```

Comment:
Remove "json" as you have it do authorized_keys as well

Reply:
ok